ocp_resources/pipelineruns.py

```diff
@@ -0,0 +1,59 @@
+# API reference: https://tekton.dev/docs/pipelines/pipelineruns/
+
+from ocp_resources.resource import NamespacedResource
+
+
+class PipelineRun(NamespacedResource):
+    api_group = NamespacedResource.ApiGroup.TEKTON_DEV
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        pipelineref=None,
+        params=None,
+        service_account_name=None,
+        client=None,
+        yaml_file=None,
+        **kwargs,
+    ):
+        """
+        Args:
+            name (str): Name of the pipelinerun
+            namespace (str): Namespace of the pipelinerun
+            pipelineref (str): Mandatory: Base pipeline to run pipelineruns
+            client: (DynamicClient): DynamicClient to use.
+            params (dict): Optional params to add during triggering a run.
```

Comment:
Add an explanation about what the dict should hold and what it will be converted to ({"name": key, "value": value})

Reply:
I don't wanted to give reference to downstream params here because those are very limited to a particular use case. These params depends on the pipelinerefernce that we gave. 

so just mentioning like: params={"key1":"value1", "key2":"value2"} and adding link to https://tekton.dev/docs/pipelines/pipelineruns/#specifying-parameters

Reply:
this is not "key1", "key2" but {param_name: param_value}

Reply:
ack. Fixed